src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -1123,6 +1130,8 @@ public void sendLastNEndpointsChangeEvent(
             forwardedEndpoints, endpointsEnteringLastN, conferenceEndpoints);
 
         sendMessage(msg);
+
+        notifyForwardedEndpointsChanged(forwardedEndpoints, endpointsEnteringLastN, conferenceEndpoints);
```

Comment:
Hm, wondering if this feels a bit weird since this call already came from _outside_ the endpoint.  If anything, it seems like we shouldn't forward this here and instead have BitrateController be the source of truth (like in your other PR)

Reply:
@bbaldino ah, very good point! Then I'll bring back event from bitrate controller and move message sending and forwarding event into event hander of bitrate controller.

Reply:
@bbaldino I think I've fixed an issues you've raising during previous review. Could you please give another chance for this PR and have a look at it? Thanks a lot!